ckan/views/group.py

```diff
@@ -827,6 +827,47 @@ def post(self, id, group_type, is_organization, data=None):
         return h.redirect_to(u'{}.bulk_process'.format(group_type), id=id)
 
 
+def changes(id, group_type, is_organization):
+    '''
+    Shows the changes to an organization in one particular activity stream item.
+    '''
+    activity_id = id
+    context = {
+        u'model': model, u'session': model.Session,
+       u'user': g.user, u'auth_user_obj': g.userobj
+    }
+    try:
+        activity_diff = get_action(u'activity_diff')(
+            context, {u'id': activity_id, u'object_type': u'group',
+                      u'diff_type': u'html'})
+    except NotFound as e:
+        log.info(u'Activity not found: {} - {}'.format(str(e), activity_id))
+       return base.abort(404, _(u'Activity not found'))
```

Comment:
The return line is wrongly indentended.
```suggestion
        return base.abort(404, _(u'Activity not found'))
```

Reply:
@Zharktas , I am sorry for this wrong indentation. I have corrected it in commit https://github.com/ckan/ckan/pull/5912/commits/57a3b50ec37e9a70c8ae145a9905e04a1324d21d